code/mobility-services/mobility-services.js

```diff
@@ -1,11 +1,25 @@
+/***************************************/
+/***** Print Menu Button ************/
+/***************************************/
+function printMenuButton(view_id) {
+  $(document).on('knack-view-render.' + view_id, function(event, view, data) {
+      $('#' + view_id + ' .knMenuLink').click(function(e) {
+        window.print();
+      });
+  });
+}
+
+printMenuButton('view_227'); // Print 3 pages
+printMenuButton('view_228'); // Print 4 pages
+
```

Comment:
Lines 12 & 13 are not wrapped in the printMenuButton function or a handler, this doesnt seem correct.

Additionally, I would suggest adding new code to the bottom of your JS/CSS files unless it already exists. This will keep code across apps in similar positions and make it easier to see what is new in a commit. New code if incorrect, would also break the rest of your codepage if placed at the top.

Reply:
moving it to line 89!